content/memmgmt_intro.tex

```diff
@@ -1,23 +1,43 @@
-\openshmem provides a set of \acp{API} for managing the symmetric heap. The
+\openshmem provides a set of \acp{API} for managing symmetric heaps. The
 \acp{API} allow one to dynamically allocate, deallocate, reallocate, and align
-symmetric data objects in the symmetric heap.
+symmetric data objects in these heaps.
 
-The symmetric memory allocation routines differ from the private heap
+In addition to the traditional (legacy) symmetric heap used by
```

Comment:
I don't think we use the term "legacy" elsewhere in the spec, e.g. to refer to operations that use the default context. Unless they are being deprecated, I would suggest that we don't introduce this term.